src/components/HomePage/FAQSection.js

```diff
@@ -4,49 +4,58 @@ import { Typography, Link } from '@material-ui/core';
 import Container from '@material-ui/core/Container';
 import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
 import MuiExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
-import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import MuiExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
 
 import Add from '@material-ui/icons/Add';
 import Remove from '@material-ui/icons/Remove';
 
 const ExpansionPanel = withStyles(theme => ({
 	root: {
 		backgroundColor: theme.palette.background.default,
-		borderTop: '1px solid rgba(0, 0, 0, 1)',
-		borderBottom: '1px solid rgba(0, 0, 0, 1)',
+		borderTop: '1px solid black',
+		borderBottom: '1px solid black',
 		boxShadow: 'none',
 		'&:not(:last-child)': {
 			borderBottom: 0
 		},
-		'&:before': {
+		'&::before': {
```

Comment:
`::before` is a [pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) which ordinarily are prefixed by `::`. `:before` is a compatibility syntax, as a single `:` refers to [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/pseudo-classes).

Reply:
Oops, thanks for pointing this out!